docs/learn/learn-extrinsics.md

```diff
@@ -0,0 +1,73 @@
+
+## Importance of Verifying Extrinsics
+
+:::info Walk-through Video Tutorial 
+
+Visit the [dedicated support page](https://support.polkadot.network/support/solutions/articles/65000179161-how-can-i-verify-what-extrinsic-i-m-signing-) and see [this technical explainer](https://youtu.be/bxMs-9fBtFk) tutorial to learn about how to verify extrinsics before signing them. The video will also mention potential attacks that can happen to you while signing for transactions. **Do not sign any transaction if you suspect you are or have been under attack.**
+
+:::
+
+:::warning Always check for Metadata Updates
+
+Before signing extrinsics with the Polkadot-JS Browser Extension and Parity Signer, always check for metadata updates. [This technical explainer video](https://youtu.be/gbvrHzr4EDY) will explain how to do it. Failing to update metadata will result in errors while signing that for example will ultimately result in you not being able to transfer funds. Before authorizing the metadata update check who is requesting it. Any request not coming from the Polkadot-JS Apps must not be signed.
+
+:::
+
+Verifying the extrinsic you are signing can take some more time before signing for a transaction but it allows you to add an extra security step. There are a multitude of possible attacks that will prevent you to send funds to the desired destination account. Here below we will list some major attacks that can happen to you ordered from the most to the least probable.
+
+### Clipboard Memory Attack
+
+:::info
+
+For a more detailed read about clipboard memory attacks see [this article](https://www.kaspersky.com/blog/cryptoshuffler-bitcoin-stealer/19976/).
+
+:::
+
+The clipboard memory is that memory on you computer dedicated to copy-paste operations. There is malicious software that can be remotely installed on your computer and that is able to detect when a cryptocurrency address is copied. Let's say you want to send funds to Address A (belonging to you), after copying address A the software swaps that address with Address B (belonging to the attacker). If you do not check the receiver address before signing the funds will be lost. 
+
+### Corrupted Website
+
+An attack can still happen if your extension is genuine, but the website (dApp) it is connected with is corrupted. In this scenario you want to perform Extrinsic A on the website, which will send Extrinsic B to the extension for signature. If you are using a Ledger device, it will display Extrinsic B. In this case it is more difficult to spot the attack and it is up to the user to know the extrinsic that was created on the website in the first place.
+
+### Corrupted QR-code (Parity Signer)
+
+If you are using [Parity Signer](https://www.parity.io/technologies/signer/), there might be the risk of scanning corrupted QR codes to sign for transactions or to update matadata. In this scenario the Signer App will display Extrinsic B when you want to sign for Extrinsic A.
```

Comment:
These are two different scenarios. If the metadata is correct, when you scan a QR to sign an extrinsic you'll see the real extrinsic info and can verify if it's the intended one. If the metadata is incorrect (or the singer you've installed is corrupted), then you run the risk of signing sth you didn't intend with no way to verify it.